pkg/iam/iam_test.go

```diff
@@ -0,0 +1,103 @@
+// Package iam checks that the AWS user currently logged in
+// has sufficient permissions to run the installer for both
+// `create` and `destroy` operations.
```

Comment:
You only need these package docs on one file in the package.  You already have it on the main `pkg/iam/iam.go`, so you can drop this comment here.

Reply:
Thanks, this is something my IDE made, that I thought must be the standard. But I looked it up and you're right. I've removed it. I'll push that out shortly.